src/txdb.h

```diff
@@ -17,8 +17,19 @@
 class CBlockFileInfo;
 class CBlockIndex;
 struct CDiskTxPos;
+struct CAddressUnspentKey;
+struct CAddressUnspentValue;
+struct CAddressIndexKey;
+struct CAddressIndexIteratorKey;
+struct CAddressIndexIteratorHeightKey;
 class uint256;
 
+typedef std::pair<CAddressUnspentKey, CAddressUnspentValue> CAddressUnspent;
+typedef std::pair<CAddressIndexKey, CAmount> CAddressAmount;
+
+typedef std::vector<CAddressUnspent> CAddressUnspentVec;
+typedef std::vector<CAddressAmount> CAddressAmountVec;
```

Comment:
I added the above 4 typedefs since they were repeated many times. Suggestions for better names would be appreciated!